app/models/press-release.js

```diff
@@ -16,4 +16,5 @@ export default class PressReleaseModel extends Model {
   @hasMany('contact') sources;
   @hasMany('file') attachments;
   @hasMany('publication-channel') publicationChannels;
+  @hasMany('government-field') governmentField;
```

Comment:
Deze relatie komt overeen met https://github.com/kanselarij-vlaanderen/app-persberichten/blob/master/config/resources/press-releases.json#L87 in de backend. De name moet dus in meervoud zijn. Ik ga deze zelf fixen zodat de PR door kan.